tests/phpunit/tests/functions/isSerialized.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+/**
+ * Tests for `is_serialized()`.
+ *
+ * @ticket 53299
+ *
+ * @group functions.php
+ * @covers ::is_serialized
+ */
+class Tests_Functions_IsSerialized extends WP_UnitTestCase {
+
+	/**
+	 * Data provider method for testing `is_serialized()`.
+	 *
+	 * @return array
+	 */
+	public function _is_serialized() {
+		return array(
+
+			// pass array, not a string.
+			array( array(), false ),
+
+			// pass a class, not a string.
+			array( new stdClass(), false ),
+
+			// pass an integer, not a string.
+			array( 0, false ),
+
```

Comment:
I would also suggest datasets with `true`, `false`, `1` (int), `0.0` (float) and `1.0` (float), all expecting a `false` return value, to ensure that truthy/falsy checks are never implemented by mistake.